.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: node_js
+os:
+  - linux
+  - osx
```

Comment:
what's missing from adding windows here?

Reply:
https://github.com/near-examples/token-contract-as/issues/12
Willem has this in progress now: https://github.com/near/devx/issues/101